src/pkg/bundle/create.go

```diff
@@ -39,7 +39,7 @@ func (b *Bundler) Create() error {
 	defer os.Chdir(cwd)
 
 	// read the bundle's metadata into memory
-	if err := utils.ReadYaml(config.BundleYAML, &b.bundle); err != nil {
+	if err := utils.ReadYaml(b.cfg.CreateOpts.UserProvidedBundleYamlFile, &b.bundle); err != nil {
```

Comment:
`UserProvidedBundleYamlFile` feels a bit verbose as a name. All bundle files are user provided, recommended shortening to something like `BundleFile`

Reply:
I have irrational flashbacks digging through old fortran code with cryptic single letter variables that I over compensate for. Sorry about that.